Looks like releases parsing is broken for ungoogled-software/ungoogled-chromium-macos

I tried to setup notifications for a new macos releases of ungoogled chromium and extension says that last published build was 199 days ago

https://github.com/ungoogled-software/ungoogled-chromium-macos/releases

I've checked extensions logs and found:

```
service-worker.js:105 Error: Release asset does not contain a valid arch
    at service-worker.js:41:15
    at Array.forEach (<anonymous>)
    at Object.ensureCompatibleReleaseFormat (service-worker.js:27:20)
    at Object.fetchReleases (service-worker.js:103:17)
    at async Object.checkForUpdates (service-worker.js:125:31)
    at async Object.update (service-worker.js:158:21)
    at async service-worker.js:185:5 
``` 

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'md5')
    at repositories.js:80:30
    at Array.map (<anonymous>)
    at repositories.js:71:30
    at Object.githubReleaseDefaultMapper (support.js:79:20)
    at ungoogled-software/ungoogled-chromium-macos (repositories.js:68:20)
    at Object.fetchReleases (service-worker.js:100:35)
    at async Object.checkForUpdates (service-worker.js:125:31)
    at async Object.update (service-worker.js:158:21)
    at async service-worker.js:185:5
```

--- a/public/repositories.js
+++ b/public/repositories.js
@@ -60,8 +60,8 @@ export default {
     }))
   },
   'ungoogled-software/ungoogled-chromium-macos': async (release) => {
-    const filenamePattern = /ungoogled-chromium[_-](?<version>\d+(?:[.]\d+)+)-(?<revision>\d+)\.(?<package_revision>\d+)_(?<arch>x86-64|arm64)-macos\.dmg/
-    const hashesPattern = /disk image `(?<name>ungoogled-chromium_.*)`: \n\n```\nmd5: (?<md5>.*)\nsha1: (?<sha1>.*)\nsha256: (?<sha256>.*)\n```/g
+    const filenamePattern = /ungoogled-chromium[_-](?<version>\d+(?:[.]\d+)+)-(?<revision>\d+)\.(?<package_revision>\d+)_(?<arch>x86[_-]64|arm64)-macos\.dmg/
+    const hashesPattern = /disk image `(?<name>ungoogled-chromium_.*?)`:[\s\S]*?```[\s\S]*?md5: (?<md5>.*)[\s\S]*?sha1: (?<sha1>.*)[\s\S]*?sha256: (?<sha256>.*)[\s\S]*?```/g
 
     const hashes = HELPERS.extractMany(release.body, hashesPattern)
 
@@ -74,7 +74,7 @@ export default {
 
         return {
           ...asset,
-          arch: assetDetails.arch,
+          arch: assetDetails.arch.replace('_', '-'),
           os: ENUMS.OS.macos,
           hashes: {
             md5: assetHashes.md5,

